ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -77,9 +76,9 @@ class PostBindingVerifier(httpResponse: Response) : BindingVerifier(httpResponse
      * Checks POST-specific rules from SSO profile spec
      * 4.1.4.5 POST-Specific Processing Rules
      */
-    private fun verifyPostSSO(samlResponseDom: Node) {
+    fun verifyPostSSO(samlResponseDom: Node) {
         if (samlResponseDom.children(SIGNATURE).isEmpty()
-                || samlResponseDom.children(ASSERTION).any {
```

Comment:
The spec says the entire response must be signed or each assertion to be signed

Reply:
```
If the HTTP POST binding is used to deliver the <Response>, each assertion MUST 
be protected by a digital signature. This can be accomplished by signing each 
individual element or by <Assertion> signing the <Response> element.
```